packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -27,6 +28,18 @@ const normalizeContentValue = (value: string) => {
   return value;
 };
 
+/**
+ * Picks identifier (`mixin`, `mixin1`, `mixin2`) from call expression as given in below example
+ * `css={{ ':hover': mixin(), border: sizes.mixin1(), ...mixin2(), ...mixin3 }}` or `css={\`\${mixin()}\`}`
```

Comment:
can we reword this? what is the input outputs?

if given `mixin()` will return `mixin` or ?